cmd/node.go

```diff
@@ -202,13 +202,15 @@ func (node *Node) Start(exitChannels ...*chan struct{}) {
 	// Setup eventManager
 	eventManager := lib.NewEventManager()
 
+	blsKeystore, err := lib.NewBLSKeystore(node.Config.PosValidatorSeed)
```

Comment:
We need to handle the error here! I'm assuming we hit an error if the seed is invalid.
```suggestion
	blsKeystore, err := lib.NewBLSKeystore(node.Config.PosValidatorSeed)
	if err != nil {
		panic(err)
	}
```